test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/enrolment/PptReferenceControllerSpec.scala

```diff
@@ -45,7 +45,7 @@ class PptReferenceControllerSpec extends ControllerSpec {
 
   private val controller = new PptReferenceController(mockAuthAction, mcc, repository, page)
 
-  val pptReference     = PptReference("XAPPT000123456")
+  val pptReference     = PptReference("XAPPT0001234567")
```

Comment:
Did the previous value not pass validation?  Does the reference now have to have an additional character at the end?

If so, do we need to update the content and TamperMonkey for this page? 
![image](https://user-images.githubusercontent.com/46934286/139839923-79c3edb5-3ecd-4d66-9c91-a82f1b00170e.png)

Reply:
Yes and yes and yes.